package.json

```diff
@@ -19,5 +19,11 @@
     "package:storeadmin": "lerna run --scope storeadmin --stream dev",
     "dev": "run-p --print-label package:*",
     "build": "lerna run --parallel build"
+  },
+  "dependencies": {
+    "@nestjs/swagger": "^5.2.1",
+    "@types/uniqid": "^5.3.2",
```

Comment:
we wont use this package. it has some problems.

Reply:
removed this dependencies package.